lib/liftoff/cli.rb

```diff
@@ -35,7 +35,7 @@ def global_options
           @options[:strict_prompts] = strict_prompts
         end
 
-        opts.on('--dependency-managers [NAME(s)]', 'Comma separated list of dependency managers to enable. Available options: cocoapods,carthage') do |list|
+        opts.on('--dependency-managers [NAME(s)]', 'Comma separated list of dependency managers to enable. Available options: cocoapods,carthage,bundler') do |list|
```

Comment:
![The Simpsons: Bart and Principal Skinner are walking along a field and a large K drops from the sky](http://dl.dropboxusercontent.com/u/3635564/gifs/k-simpsons.gif)